tdrs-backend/tdpservice/settings/common.py

```diff
@@ -228,30 +233,35 @@ class Common(Configuration):
                 "class": "logging.StreamHandler",
                 "formatter": "color",
             },
+            "file": {
+                "class": "logging.FileHandler",
+                "formatter": "verbose",
+                "filename": '/logs/django.log',
+            }
         },
         "loggers": {
             "tdpservice": {
-               "handlers": ["application"],
+               "handlers": ["application", "file"],
```

Comment:
i believe this to have broken deployments - https://app.circleci.com/pipelines/github/raft-tech/TANF-app/28966/workflows/bbe12d8d-d147-4288-8308-e9e8a6ec6e22/jobs/85249

Reply:
Indeed it has. I changed the path for the log file on my PLG deployment ticket because of this issue. I Will make a hotfix PR. Thank you for catching this!